### PR TITLE
doc(lab3): TODO from issues 23

### PR DESCRIPTION
**Pull request by ljeanner**
- [x]  Add a Little explanation on what will be done in Lab 3
- [x]  What is APIM
- [x]  Add laïus on API/products in APIM
- [x] Laïus on app registration.
- [x]  TBD : how we get the tenant ?? <= Add an instructor led instruction on this tab
- [ ] begin, we will set a limit for the Basic user to ensur <= as it represents a free usage of our api
- [ ]  Add Instructions on where to get the URL to call for the APIM policy tests

### CODE REVIEW

File: docs/workshop.md [hunk 1]
```diff
@@ -1223,12 +1225,43 @@ In this lab, you learned how to process and fetch orders using Azure Functions a
 
 # Lab 3 : Exposing and monetizing APIs (45m)
 
-For this Lab, we will focus on the following scope :
+For this Lab, we will focus on exposing and monetizing APIs using Azure API Management (APIM). The steps will guides you through the process of creating and managing APIs, setting up products, and securing the APIs using subscription keys and OAuth 2.0. The goal is to demonstrate how to publish APIs, manage access, and define usage plans to monetize the APIs effectively within the the following scope :
```
Comment (ikhemissi) [inline docs/workshop.md:1228]: there is a duplicate `the` in `within the the following scope`

File: docs/workshop.md [hunk 2]
```diff
@@ -1223,12 +1225,43 @@ In this lab, you learned how to process and fetch orders using Azure Functions a
...
+
+**Analytics and Monitoring**: Provides insights into API usage, performance, and health.
+
+**Transformation and Enrichment**: Allows modification of requests and responses, such as format conversion and data enrichment.
+
+**Policy Management**: Enables the application of policies to APIs for security, caching, transformation, and more.
+
+<div class="info" data-title="Note">
+
+> You can find a detailed article how APIM helps organizations expose their services as APIs, manage access, enforce policies, and gain insights into API usage, ultimately enabling them to monetize their APIs effectively. [following this link](https://learn.microsoft.com/en-us/azure/api-management/api-management-key-concepts).
```
Comment (ikhemissi) [inline docs/workshop.md:1253]: There are no mentions to monetization in this article, it is the overview page which introduces the basic concepts of APIM :-(

File: docs/workshop.md [hunk 3]
```diff
@@ -1402,6 +1440,12 @@ We will now see how to secure our API with the OAuth 2.0 standard
 
 In this section, we will learn how to configure EntraId to enable OAuth security using your identity provider, Azure Entra ID. We will create two App Registrations: one representing the API in Azure Entra ID and another representing the API caller.
 
+<div class="info" data-title="Note">
+
+> An app registration in Azure Active Directory (Azure AD) is a process that allows you to integrate your application with Azure AD to enable authentication and authorization. By registering an application, you create an identity for it, which Azure AD can use to authenticate users and grant access to resources.
```
Comment (ikhemissi) [inline docs/workshop.md:1445]: We should use Entra ID in this paragraph instead of Azure Active Directory

File: docs/workshop.md (hunk 2 above)
Comment (ljeanner) [reply docs/workshop.md:1253]: image
a la fin du lab 3

Comment (ikhemissi) [reply docs/workshop.md:1238]: I meant the article https://learn.microsoft.com/en-us/azure/api-management/api-management-key-concepts in the paragraph https://github.com/ikhemissi/azure-ipaas-workshop/pull/27/files#diff-d88cdc3661331e4b78f6f54532457d2116e62ae6a491f1750818b6612dc0a6bcR1238

**Pull request merged**